app/models/log-message.js

```diff
@@ -0,0 +1,9 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
```

Comment:
Yeah, we could do that. Loggly kinda sucks, but would probably be more than enough for our purposes at the moment. Maybe i'll put together another PR for something like that? This was more/less just to debug really crazy stuff at the moment.